### ISSUE TITLE
Separate routes for each lesson

### ISSUE THREAD

**Issue by julianorinyol** [rdlauer/nslayouts#6]
It would be nice to be able to link to a specific lesson, so that i can refer back to it later, or send it to someone else. 
perhaps there could be a  *lesson* query parameter ie. `?lesson=2`**  to make this possible.

and when `Next` button is clicked to proceed to the next lesson, it would change this route as well.

My specific use case, is that I wanted to link to the lesson 16 with info on the rows="2*, *" syntax from notes i'm taking, in case i need a refresher later.

**Comment by rdlauer** [comment]
Makes sense. You can also navigate directly to a lesson by choosing it from the dropdown provided:

![image](https://user-images.githubusercontent.com/1918280/61637624-8718db80-ac5d-11e9-80ed-9407ef8dd6c3.png)
